projects/github.com/xiph/rav1e/package.yml

```diff
@@ -0,0 +1,27 @@
+distributable:
+   url: https://github.com/xiph/rav1e/archive/refs/tags/v{{ version }}.tar.gz
+   strip-components: 1
+
+versions:
+  github: xiph/rav1e/tags  # reads github tags from github
+  strip: /^(?!v).*/
```

Comment:
Hmm, what's this about?

Reply:
Some of the pre-release tags have p<date> like p20230101.  They were breaking the build, so I excluded them when pulling versions.  If we wanted to include them, how would we do so?

Reply:
`versions.ignore: /^p/`

Reply:
Oh, sorry, you want to _include_ them? `versions.strip: /^p/`.